src/test/java/org/jenkinsci/plugins/android_lint/LintResultTest.java

```diff
@@ -25,7 +25,7 @@
     public void testResultDuplicates() throws ExecutionException, InterruptedException, IOException {
         // Create a project with a Lint results file in its workspace.
         FreeStyleProject project = jenkins.createFreeStyleProject();
-        project.setScm(new ExtractResourceSCM(getClass().getResource("duplicates.zip")));
+        project.setScm(new ExtractResourceSCM(getClass().getResource("parser/duplicates.zip")));
```

Comment:
@orrc I did this changes in order to apply the same criteria that in other analysis plugins. Reviewed.